vultr.php

```diff
@@ -2214,22 +2085,21 @@ public function tabClientActions($package, $service, array $get = null, array $p
     }
 
     /**
-     * Client Stats tab.
+     * Actions (Admin or Client) tab.
      *
      * @param stdClass $package A stdClass object representing the current package
      * @param stdClass $service A stdClass object representing the current service
-     * @param array $get Any GET parameters
      * @param array $post Any POST parameters
-     * @param array $files Any FILES parameters
+     * @param string $template_name The name of the template to us
      * @return string The string representing the contents of this tab
      */
-    public function tabClientStats($package, $service, array $get = null, array $post = null, array $files = null)
+    private function getTabActions($package, $service, array $post = null, $template_name = 'tab_actions')
```

Comment:
It may be better to pass a boolean value of whether to load the client template or not, rather than specifying the exact tab to load, e.g.

 private function getTabActions($package, $service, array $post = null, $client = false)
{
    $template = ($client ? 'tab_client_actions' : 'tab_actions');
    $this->view = new View($template, 'default');

    ...
}

Reply:
Sure, I suppose there isn't really a chance of their being more than two options.